bazel/go_proto_library.bzl

```diff
@@ -326,7 +330,7 @@ def go_proto_library(name, srcs = None, deps = None,
   go_library(
       name = name,
       srcs = [":" + name + _PROTOS_SUFFIX],
-      deps = deps + grpc_deps + [golang_protobuf],
```

Comment:
As an aside, it seems to go against the spirit of the original `go_library` to force inclusion of WKTs. I think this would be better placed in https://github.com/lyft/protoc-gen-validate/blob/master/bazel/pgv_proto_library.bzl#L4.

Reply:
I would like that too, but the go_proto_library rule used to implement the pgv_proto_library rule requires that the listed deps are proto libraries (by appending "_proto" to the names). `com_github_golang_protobuf//ptypes:go_default_library` is a go library, though, so it can't be listed as a dependency. I could condition the WKT inclusion on the `validate` flag?

Reply:
+1